charts/airbyte/templates/env-configmap.yaml

```diff
@@ -61,7 +61,7 @@ data:
   TEMPORAL_HOST: {{ .Release.Name }}-temporal:{{ .Values.temporal.service.port }}
   TEMPORAL_WORKER_PORTS: 9001,9002,9003,9004,9005,9006,9007,9008,9009,9010,9011,9012,9013,9014,9015,9016,9017,9018,9019,9020,9021,9022,9023,9024,9025,9026,9027,9028,9029,9030,9031,9032,9033,9034,9035,9036,9037,9038,9039,9040
   TRACKING_STRATEGY: segment
-  WEBAPP_URL: http://{{ .Release.Name }}-airbyte-webapp-svc:{{ .Values.webapp.service.port }}
+  WEBAPP_URL: {{ if .Values.webapp.url }}{{ .Values.webapp.url }}{{ else }}http://{{ .Release.Name }}-airbyte-webapp-svc:{{ .Values.webapp.service.port }}{{ end }}
```

Comment:
@AndreaGiardini can you change this to use [default](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) instead of an inline if-else? That'll be more conventional I think

Reply:
@pmossman It should be fixed now. Can you have a second look, please?

Reply:
Thanks for making that change, I will open an internal pull request where I can validate the change further